### ISSUE TITLE
Readd original repo

### ISSUE THREAD

**Issue by Arcoz0308** [Vecnavium/FormsUI#2]
Please readd in the readme that this are a fork of FormAPI

**Comment by Vecnavium** [comment]
Okay and what's the point of creating such an issue? 
Most of the community already knows this is a fork of FormAPI and im the one that decided to maintain FormAPI for PocketMine API 4.0.0 and up. We even have a discussion last night about the issues FormAPI has right now and I stud up saying I will maintain FormAPI and patch the issues FormAPI on my own version in FormsUI as this is not for PM3 its for PM4.
I have nothing to hide and I am admitting this is a FormAPI fork and so does the PM Community know, so you creating such issue isn't needed unless you are actually trying to point out something then maybe reword it, if not,
Thank you.

**Comment by Arcoz0308** [comment]
The license of FormAPI say that for each fork you need to add the original link, yes many poeple know that are a fork of FormAPI but not everyone

Please respect the license of FormAPI
